### PR TITLE
✨  Add ReferenceList

### LINKED ISSUE #9: Episode Reference Page

### PR DESCRIPTION
**Pull request by christophvogel**
✨ Add ReferenceList
✨ Add back button to EpisodeInfo
✨ Add button svg
✨ Update episode page
💄 Update globals.css

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
GJ! 🚀

**Review by SilviaBarcellini — APPROVED** [review]
looks good! 👍

File: components/EpisodeInfo/EpisodeInfo.tsx [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 export type EpisodeProps = {
   episodeName: string;
   imgSrc: string;
 };
 
 function EpisodeInfo({ episodeName, imgSrc }: EpisodeProps) {
+  const router = useRouter();
   return (
     <div className={styles.info_container}>
+      <img
```
Comment (lmachens) [inline components/EpisodeInfo/EpisodeInfo.tsx]: If this is only used for navigation, better use a `<Link>`/`<a>` here. 
https://nextjs.org/docs/api-reference/next/link

Same issue in `ReferenceList`

File: components/ReferenceList/ReferenceList.module.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.container {
+  background-color: var(--primary-component-color);
+  border-radius: 10px;
+  margin: 10%;
+  padding: 2%;
+  height: 35vh;
```
Comment (lmachens) [inline components/ReferenceList/ReferenceList.module.css:6]: It's not recommended to set height or width relative to the viewport for components. Components should be usable in different context, thus, it's up to the parent container to limit the size.
Same with the `margin`.

**Review by lmachens — COMMENTED** [review]
Good job

**Pull request merged**